src/systems/ec/implementations/vanilla/packages/ec-component-table/ec-component-table.scss

```diff
@@ -162,6 +162,47 @@
       padding-top: $ecl-spacing-s;
     }
   }
+
+  // SortTable
+  .ecl-table__header[data-ecl-table-sort-toggle] {
+    cursor: pointer;
+    padding-left: calc(
+      #{$ecl-spacing-m} + #{$ecl-icon-2-xs} + #{$ecl-spacing-xs}
+    );
+    position: relative;
+  }
+
+  .ecl-table__arrow {
+    left: $ecl-spacing-m;
+    position: absolute;
```

Comment:
Isn't this parent element of absolute children supposed to be relative to hold them?

Reply:
parent container can be _absolute_ if needed.